SRP/Example4.md

```diff
@@ -0,0 +1,23 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/keycloak/keycloak/blob/89795cfd7df24aad392c6223fe6d4639f03bd03d/core/src/main/java/org/keycloak/TokenIdGenerator.java#L27
+
+#### Why
+
+Класс занимается только генерацией id
+
+------------------------
+
+### Violation
+
+---
+#### Where
+
+https://github.com/keycloak/keycloak/blob/89795cfd7df24aad392c6223fe6d4639f03bd03d/core/src/main/java/org/keycloak/TokenVerifier.java#L44
```

Comment:
➕
I didn't get why we should keep inside the TokenVerifier class a lot of other classes.